cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -40,11 +40,12 @@ type MasterConfiguration struct {
 	KubernetesVersion string
 	CloudProvider     string
 	AuthorizationMode string
+	CertAltNames      []string
+	CertificatesDir   string
```

Comment:
"The directory that will hold all of the generated certificates and private keys."